src/main/java/com/ibanity/apis/client/paging/IbanityPagingSpec.java

```diff
@@ -1,107 +1,23 @@
 package com.ibanity.apis.client.paging;
 
-import io.crnk.core.queryspec.pagingspec.OffsetLimitPagingSpec;
-import io.crnk.core.queryspec.pagingspec.PagingSpec;
-import org.apache.commons.lang3.builder.EqualsBuilder;
-import org.apache.commons.lang3.builder.HashCodeBuilder;
-import org.apache.commons.lang3.builder.ToStringBuilder;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.UUID;
 
-public class IbanityPagingSpec extends OffsetLimitPagingSpec implements PagingSpec {
-
-    private UUID before = null;
-    private UUID after = null;
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Beware of the `@AllArgsConstructor` which may cause issues on attributes refactoring/reordering in the  java file.
Order of the args will be changed and may lead to unexpected behaviour.

Reply:
Fixed with @AllArgsConstructor(access = AccessLevel.PRIVATE)